source/scss/clarendon.scss

```diff
@@ -0,0 +1,21 @@
+/*
+ * Clarendon
+ *
+ */
+@import 'shared';
+
+%clarendon,
+.clarendon {
+  @extend %filter-base;
+  filter: contrast(1.2) saturate(1.35);
+
+  &:after {
+    background: rgba(239, 239, 239, .8);
+    mix-blend-mode: soft-light;
+  }
+
+  &:before {
```

Comment:
@una thanks this looks better! Although what do you think about `rgba(127, 187, 227, 0.2)`? Having a darker color seems more apt it seems. https://youtu.be/W-uspcrRe6Y